### PR TITLE
refactor puzzle and round creation

### PR DESCRIPTION
**Pull request by rdren0**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "mocha": "^5.2.0",
     "mochapack": "^1.0.0",
     "p5": "^0.7.3",
+    "sass": "^1.17.3",
```
Comment (misskelly) [inline package.json]: 😍

File: src/Game.js [hunk 2]
```diff
@@ -16,39 +15,31 @@ class Game {
...
-      this.round = new Round(players)
-      puzzle.puzzleSet(data.puzzles.three_word_answers);
-  }   else{
-    this.round = new Round(players)
-    puzzle.puzzleSet(data.puzzles.four_word_answers);
-
+    if (round === 4){
+      // bonuswheel
+    }
+    this.round = new Round(players,this.allData[this.round-1]);
```
Comment (misskelly) [inline src/Game.js]: This is super awesome.

File: src/Puzzle.js [hunk 3]
```diff
@@ -1,31 +1,23 @@
...
 // import domUpdates from "./domUpdates.js";
 
 class Puzzle {
-  constructor() {
+  constructor(puzzleArry) {
 
   this.hint = null;
-  this.puzzleobj = null;
+  this.puzzleArray = [];
+  this.puzzleobj = {};
```
Comment (misskelly) [inline src/Puzzle.js]: My understanding was that it's not considered best practice to use data types in variable names.  At some point can we double check with an instructor on that?

**Comment by rdren0** [comment]
@maxbsilver

File: src/Round.js [hunk 4]
```diff
@@ -1,15 +1,27 @@
...
-// import domUpdates from "./domUpdates.js";
+import Player from "./Player.js";
+import Puzzle from "./Puzzle.js";
+import domUpdates from "./domUpdates.js";
 
 class Round {
-  constructor() {
+  constructor(players, bank) {
     this.players = players;
+    this.bank = []
```
Comment (misskelly) [inline src/Round.js]: Maybe we could rename this.bank to this.puzzleBank for clarity?

**Comment by rdren0** [comment]
@Rosebud303

**Pull request merged**